app/controllers/vendor_api/test_data_controller.rb

```diff
@@ -67,5 +68,9 @@ def for_ratified_courses_param
     def for_test_provider_courses_param
       params[:for_test_provider_courses] == 'true'
     end
+
+    def previous_cycle_param
```

Comment:
Can we use a more idiomatic method name like `previous_cycle?` as this returns a boolean based on the value of the param, and not the param value itself.

Reply:
Thanks, fixed